tts/sapi.py

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env python3
 """
-This ia a helper to use the Microsoft SAPI
+This is a helper to use the Microsoft SAPI
 
-Needs to run on a windows system
+Needs to run on a Windows system
 Requires comtypes
 """
 
+from os import name # Allows checking if using Windows
```

Comment:
Don't pollute the namespace with a commonly used variable. Instead import `os` and use `os.name`

Reply:
fixed